web/src/App.js

```diff
@@ -1,20 +1,19 @@
 import Login from "./pages/Login";
-import { Welcome } from "./pages/Welcome";
+import Welcome from "./pages/Welcome";
 import { BrowserRouter, Routes, Route } from "react-router-dom";
-import "./App.css"
+import "./App.css";
 
-function App() {
+const App = () => {
   return (
     <div className="App">
       <BrowserRouter>
         <Routes>
-          <Route path="*" element={<Login />} />
           <Route path="/login" element={<Login />} />
```

Comment:
what happens if we goto localhost:3000 now?

Reply:
Display an empty window in browser

Reply:
It should be redirected to login page, can you please do that change

Reply:
Changes are done and push to the branch

Reply:
we need to redirect to /login not just show login component